README.md

```diff
@@ -1 +1,116 @@
-# hello-rails-back-end
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#About-the-Project)
+- [📖 React Repo Link](#React-repo-link)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [Setup](#setup)
+- [👥 Authors](#Authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#Show-your-support)
+- [🙏 Acknowledgements](#Acknowledgments)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# [About the Project] <a name="Greetings"></a>
+
+**[Greetings]** The Greetings App is a web application built using React, Redux, and Ruby on Rails. It provides a user-friendly interface to simulate a greetings program.
+
+# [React Repo Link] <a name="React Repo Link"></a>
+https://github.com/ismailmunyentwari9/hello-react-front-end.git
+
+##  Built With <a name="RoR&React_Redux"></a>
+### Tech Stack <a name="tech-Stack"></a>
+- **[RUBY]**
+- **[RAILS]**
+- **[REACT]**
+- **[REDUX]**
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/RUBY">RUBY</a></li>
+  </ul>
+</details>
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/RAILS">RAILS</a></li>
+  </ul>
+</details>
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/REACT">REACT</a></li>
+  </ul>
+</details>
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/REDUX">REDUX</a></li>
+  </ul>
+</details>
```

Comment:
I notice that all the links in the `Tech Stack` is broken, kindly fix this please, let's avoid broken links in our README for a professional repository 🤗